tests/study_tests/test_study.py

```diff
@@ -1577,3 +1578,24 @@ def test_set_invalid_metric_names() -> None:
     study = create_study(directions=["minimize", "minimize"])
     with pytest.raises(ValueError):
         study.set_metric_names(metric_names)
+
+def objective(trial):
+    x = trial.suggest_uniform('x', -10, 10)
+    y = trial.suggest_uniform('y', -5, 5)
+    z = trial.suggest_uniform('z', 0, 1)
```

Comment:
Optuna repo has introduced a few packages to unify the coding style as explained in https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md. Part of these errors would be removed by running `./formats.sh`.

Reply:
Hi, Do you have a Linkedin? Wanted to connect on knowing more about the library so that i can contribute better

Reply:
Mine or Optuna?

Reply:
Yours too. And if Optuna has a slack channel or something, that would be helpful too

Reply:
Optuna does not have any open slack channel. Sorry for the thin communication, I hope we can communicate via GitHub. If you have any concerns, please create an issue or discussion!